gravitee-am-ui/src/app/domain/settings/password-policy/domain-password-policy.component.ts

```diff
@@ -126,4 +140,84 @@ export class DomainPasswordPolicyComponent implements OnInit {
     delete policy['updatedAt'];
     this.passwordPolicy = { ...policy };
   }
+
+  private getIdentityProviderDetails(type: string) {
+    const identities = this.route.snapshot.data['identities'];
+    if (identities && identities[type]) {
+      return identities[type];
+    }
+  }
+
+  private getAssociatedPolicyName(id: string): string {
+    const policies = this.route.snapshot.data['policies'];
+    return policies.find((pp) => pp.id === id)?.name;
+  }
+
+  private updateIdps() {
+    if (this.idpToUpdate !== undefined) {
+      this.idpToUpdate.idpsToLink.forEach((id) => {
+        this.providerService.assignPasswordPolicy(this.domainId, id, this.policyId).subscribe(() => {
+          this.idpUpdated = false;
+        });
+      });
+      this.idpToUpdate.idpsToUnlink.forEach((id) => {
+        this.providerService.assignPasswordPolicy(this.domainId, id, undefined).subscribe(() => {
+          this.idpUpdated = false;
+        });
+      });
+    }
```

Comment:
TODO: I wondering if there is a way to merge all the observable on each request to do a single subscribe in order to update only once the idpUpdated :thinking: 

Something using forkJoin or mergeAll ? 

Also if we can have this "single" subscription it could be nice to call it into the main update stream to display the snackbar once all updates are done.